tests/unit-tests/api/system-status.php

```diff
@@ -250,6 +250,37 @@ public function test_get_system_tools() {
 			),
 			$data
 		);
+
+		$query_params = array(
+			'_fields' => 'id,name,nonexisting',
+		);
+		$request = new WP_REST_Request( 'GET', '/wc/v2/system_status/tools' );
```

Comment:
Everything looks good; I just think for these we should be testing the `v3` endpoints. We should probably update all of the api unit tests to use the v3 endpoints.

Reply:
Hm, that's a good point. Shouldn't we rather copy the files we have for testing v2 API and replace `v2` with `v3` and only then add new tests to this new file? How would we otherwise know we haven't broken anything in both v2 and v3 APIs when doing changes..?